runtime.txt

```diff
@@ -1 +0,0 @@
-python-3.13.0
```

Comment:
IIRC this was needed for heroku not for the package

Reply:
As mentioned in the previous reply, this gets generated via the buildpack on heroku side.